src/site/assets/google-analytics/vagovdev.html

```diff
@@ -0,0 +1,2 @@
+<noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-WFJWBD&gtm_auth=inC4EKQce9vlWpRVcowiyQ&gtm_preview=env-423&gtm_cookies_win=x"
+  height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
```

Comment:
I don't know if `assets` is the perfect spot for this, but I want it close to the JS snippet, and I can't put the JS snippet anywhere else or else the linter will flip.